### PR TITLE
Feature/cards

### PR DESCRIPTION
**Pull request by abarmina**

### CODE REVIEW

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.html [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
-        <div class="item__date-end">valid to {{ offer.dateEnd }}</div>
+        <div class="item__date-end">
+          {{ 'CARD.VALID_TO' | translate }} {{ offer.dateEnd }}
+        </div>
         <div class="item__actions">
           <button
             *ngIf="offer.promocode"
             mat-button
-            class="button"
+            class="button button_promocode"
```
Comment (AlexanderBazukevich) [inline src/app/pages/home-page/offer-item-page/offer-item-page.component.html]: I recommend to extract button styles to shared button and add some modificators.

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.html [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
       <div class="item__footer">
         <div class="item__rate">
-          <div class="item__text">Did you use the offer? Rate it!</div>
-          <button mat-flat-button class="button button--review">review</button>
+          <div class="item__text">{{ 'CARD.RATE' | translate }}</div>
+          <button mat-flat-button class="button button_review">review</button>
         </div>
         <div class="item__favorite">
-          <div>Add to bookmarks</div>
+          <div>{{ 'CARD.BOOKMARKS_TEXT' | translate }}</div>
```
Comment (AlexanderBazukevich) [inline src/app/pages/home-page/offer-item-page/offer-item-page.component.html:36]: In translate file you have CARD.BOOKMARK key, so english text is lost on page

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.html [hunk 3]
```diff
@@ -43,12 +42,13 @@
...
-        [routerLink]="['/home/offices', office.id]"
-      >
+
+    <mat-tab label="{{ 'CARD.ADDRESSES' | translate }}">
+      <div *ngFor="let office of offices" class="address">
+        <a
+          class="address__link-overlay link"
+          [routerLink]="['/home/offices', office.id]"
+        ></a>
         <mat-icon class="address__location-icon">location_on</mat-icon>
```
Comment (AlexanderBazukevich) [inline src/app/pages/home-page/offer-item-page/offer-item-page.component.html:54]: icon styles better get from shared icon styles file

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.scss [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
 @import '/src/app/styles/components/tabs.scss';
 @import '/src/app/styles/components/addresses.scss';
+@import '/src/app/styles/custom-theme.scss';
 
 .button {
   text-transform: uppercase;
-  color: $blue-primary;
 
-  &--review {
+  &_review {
```
Comment (AlexanderBazukevich) [inline src/app/pages/home-page/offer-item-page/offer-item-page.component.scss]: for example
white color on blue background - button_c_primary;
blue color on transparent background - button_c_default

File: src/app/shared/custom-components/language/language.component.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<button class="button button_s_l" mat-button (click)="onChangeLang()">
-  {{ 'HOME.LANG' | translate }}
+<button class="button button_l" mat-button (click)="onChangeLang()">
```
Comment (AlexanderBazukevich) [inline src/app/shared/custom-components/language/language.component.html]: please return my modificator "button_s_l" )))

**Review by AlexanderBazukevich — CHANGES_REQUESTED** [review]

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.html (hunk 2 above)
Comment (abarmina) [reply src/app/pages/home-page/offer-item-page/offer-item-page.component.html:36]: thanks, it's my mistake

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.html (hunk 3 above)
Comment (abarmina) [reply src/app/pages/home-page/offer-item-page/offer-item-page.component.html:54]: there in styles only margin. I don't know, but maybe it's just a local style?

```
  &__location-icon {
    margin-right: 0.5em;

    @media screen and (max-width: $mobile) {
      margin-right: 0.3em;
    }
  }
  ```

File: src/app/shared/custom-components/language/language.component.html (hunk 5 above)
Comment (abarmina) [reply src/app/shared/custom-components/language/language.component.html]: I'm so sorry, corrected

**Review by Zairbekov — APPROVED** [review]

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.html (hunk 1 above)
Comment (abarmina) [reply src/app/pages/home-page/offer-item-page/offer-item-page.component.html]: didn't have time then, did now

File: src/app/pages/home-page/offer-item-page/offer-item-page.component.scss (hunk 4 above)
Comment (abarmina) [reply src/app/pages/home-page/offer-item-page/offer-item-page.component.scss]: did it

**Review by AlexanderBazukevich — APPROVED** [review]

File: src/app/pages/home-page/office-item-page/office-item-page.component.html [hunk 6]
```diff
@@ -1,34 +1,55 @@
...
+      />
+      <div class="office__info">
+        <a
+          class="link link_c_default card__name office__vendor"
+          [routerLink]="['/home/vendors', office.vendorId]"
+        >
+          {{ vendor.name }}
+        </a>
+        <div class="card__title office__contact">
+          <mat-icon>location_on</mat-icon>{{ office.address }}
```
Comment (tlazarchuk) [inline src/app/pages/home-page/office-item-page/office-item-page.component.html:17]: data in <div> element? Maybe add text in <p> or <a>

File: src/app/styles/components/addresses.scss [hunk 7]
```diff
@@ -17,4 +17,18 @@
       margin-right: 0.3em;
     }
   }
+
+  &__link-overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```
Comment (tlazarchuk) [inline src/app/styles/components/addresses.scss:26]: We can make 4 lines(from 23 to 26) into 1 line?

**Review by tlazarchuk — APPROVED** [review]

File: src/app/styles/components/addresses.scss (hunk 7 above)
Comment (abarmina) [reply src/app/styles/components/addresses.scss:26]: I do not know how

**Pull request merged**